cypress/integration/appSettings.ts

```diff
@@ -2,6 +2,7 @@ import initialDBData from '../support/initialDBData';
 
 context('App settings tests', () => {
```

Comment:
Sorry for additional changes. As we've been talking about test coverage, I think these features could need integration tests.